azure-ipam/main.go

```diff
@@ -22,9 +22,10 @@ func main() {
 func executePlugin() error {
 	// logger config
 	loggerCfg := &logger.Config{
-		Level:            "debug",
-		OutputPaths:      "var/log/azure-ipam.log",
-		ErrorOutputPaths: "var/log/azure-ipam.log",
+		Level:                "debug",
+		Filepath:             "/var/log/azure-ipam.log",
+		MaxSizeInMB:          5, // MegaBytes
+		MaxBackups:           8,
```

Comment:
wondering if you did any testing to arrive at these values? I want to make sure we are not rotating the logs too aggressively - it would hard to troubleshoot if that was accidentally like, 10 minutes of logs in a decently sized cluster

Reply:
I did test that. We don't log much into this file for now. It took alot of time for me with an script to reach to 5 MB. It was @tamilmani1989 suggestion to go with these numbers for now.

Reply:
keeping same as cni log files